src/commcare_cloud/ansible/roles/pgbouncer/templates/pgbouncer_hba.conf.j2

```diff
@@ -27,8 +31,11 @@ host   {{ db.name }}	{{ db.user }}	{{ lookup('dig', host, wantlist=True)[0] }}/3
 {% endif %}
 {% endfor %}
 {% endfor %}
+{% endif %}
 
-{% for host in (groups.get('citusdb_worker', []) + groups.get('citusdb_cordinator', []) ) | unique %}
+{% set citus_host_list = groups.get('citusdb_worker', []) + groups.get('citusdb_cordinator', []) | unique %}
+{% if inventory_hostname in citus_host_list %}
+{% for host in citus_host_list %}
```

Comment:
This removed three of these sections from india pgbouncer_hba.conf when I rolled it out, which I think was the intent